src/main/kotlin/com/exactpro/th2/codec/EventProcessor.kt

```diff
@@ -99,11 +99,13 @@ class EventProcessor(
         .status(status)
         .type(if (status != Event.Status.PASSED) "Error" else "Warn")
         .bodyData(EventUtils.createMessageBean("This event contains reference to the codec event"))
-        .bodyData(ReferenceToEvent(eventId.logId))
+        .bodyData(ReferenceToEvent(eventId.idString))
         .toProto(this)
         .also(onEvent)
         .id
 
+    private val EventID.idString get() = "${bookName}:${scope}:${Timestamps.toString(startTimestamp)}:${id}"
```

Comment:
Please, take a look at `StoredTestEventIdUtils` in cradle library. The ID format should be the same

Reply:
@lumber1000 Please, correct that. The current event format won't work as a reference to event